exercises/J-functions/exercise.js

```diff
@@ -1,7 +1,10 @@
 function halve(number) {
   // complete the function here
+  return number / 2;
 }
 
 var result = halve(12);
+var results = halve(20);
```

Comment:
Be careful when naming variables very similarly, also try not to use var, use const or let if you can var is a bit more old school